### PR TITLE
updated inedex.md to fix commas and punctuation.

### PR DESCRIPTION
**Pull request by bradbaas**

### CODE REVIEW

**Review by kingthorin — COMMENTED** [review]
This is generated content, it needs to be changed elsewhere. I'll provide a link in a bit.

**Comment by kingthorin** [comment]
It'd be this file that you should PR against:

https://github.com/zaproxy/zap-core-help/blob/main/addOns/help/src/main/javahelp/contents/start/start.html

File: site/content/docs/desktop/start/_index.md [hunk 1]
```diff
@@ -7,34 +7,34 @@ weight: 2
...
 # Getting Started
 
-The quickest way to get going with ZAP is to use the Quick Start add-on, which is installed by default.  
+The quickest way to get going with ZAP is to use the Quick Start add-on which is installed by default.  
 This allows you to enter a URL which ZAP will first spider
 and then [active scan](/docs/desktop/start/features/ascan/).  
 For a more in depth test you should explore your application using your browser or automated regression tests while proxying through ZAP.
 
-At its heart ZAP is a [manipulator-in-the-middle proxy](/docs/desktop/start/features/intercept/).  
+At its heart ZAP is a [man-in-the-middle proxy](/docs/desktop/start/features/intercept/).
```
Comment (kingthorin) [inline site/content/docs/desktop/start/_index.md:15]: This was intentional

**Comment by psiinon** [comment]
Closing as per the above comments - this is the wrong file to change.

**Pull request closed without merging**